### PR TITLE
add proper message for derby related issue

### PR DESCRIPTION
**Pull request by stefanopuddu**
add just a warning instead of a full stacktrace for a known derby issue

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/ContentDAO.java [hunk 1]
```diff
@@ -701,8 +687,8 @@ private int loadContentStatus(Connection conn, String query) {
 				count = res.getInt(1);
 			}
 		} catch (Throwable t) {
-			_logger.error("Error loading contents status", t);
-			throw new RuntimeException("Error loading contents status", t);
+			_logger.error("Error loading contents status. If you are runing Entando backed by Apache Derby it's a known issue");
+			//throw new RuntimeException("Error loading contents status", t);
```
Comment (nicpuddu) [inline engine/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/ContentDAO.java]: was this intentional?
please delete the line if that was the case.

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**